app/routers/logout.py

```diff
@@ -0,0 +1,17 @@
+from fastapi import APIRouter, Request
+from starlette.responses import RedirectResponse
+from starlette.status import HTTP_302_FOUND
+
+
+router = APIRouter(
+    prefix="",
+    tags=["/logout"],
+    responses={404: {"description": "Not found"}},
+)
+
+
+@router.get('/logout')
+async def logout(request: Request):
```

Comment:
Can we somehow check the logout request using some CSRF cookie? Currently it will disconnect you if you'll click a [link](https://calendar.pythonic.guru/logout) that I've sent you.

Reply:
(Don't overthink it if you find it too hard - I do want to finish this ticket ASAP)

Reply:
I started to read about it. Seems like it's a whole topic of security, which is new to me.
I am always eager to learn more but that will take some time to understand in a proper depth.
If we can leave it as it is for now, I promise to try to improve it after i finish my tickets.
If you have any good suggestions for me regarding what on this subject to put my focus on, i will be more then happy to know.

Reply:
Sababa :)